csfieldguide/chapters/content/en/formal-languages/sections/finite-state-automata.md

```diff
@@ -525,21 +527,23 @@ Then at the bottom of the window you can trace the string one character at a tim
 If you step right through the string and end up in a final (accepting) state, the panel will come up green.
 To return to the Editor window, go to the "File" menu and select "Dismiss Tab".
 
-{image file-path="img/chapters/jflap-create-accept.png" alt="Building an FSA - testing with input"}
+{image file-path="img/chapters/jflap-create-accept.png" alt="Building an FSA &ndash; testing with input"}
 
 You can run multiple tests in one go.
 From the "Input" menu choose "Multiple Run", and type your tests into the table, or load them from a text file.
 
-{image file-path="img/chapters/jflap-create-accept-multi.png" alt="Building an FSA — multiple run"}
+{image file-path="img/chapters/jflap-create-accept-multi.png" alt="Building an FSA &ndash; multiple run"}
 
 You can even do tests with the empty string by leaving a blank line in the table, which you can do by pressing the "Enter Lambda" button.
 
 There are some FSA examples in the next section.
 If you're doing this for a report, keep copies of the automata and tests that you do (JFLAP's "File" menu has a "Save Image As..." option for taking snapshots of your work; alternatively you can save an FSA that you've created in a file to open later).
 
+{panel end}
+
 ### Examples to try
 
-Using Exorciser or JFLAP, construct an FSA that takes inputs made of the letters "a" and "b", and accepts the input if it meets one of the following requirements.
+By hand, or using Exorciser or JFLAP, construct an FSA that takes inputs made of the letters "a" and "b", and accepts the input if it meets one of the following requirements.
```

Comment:
Consider changing to 'By hand, using Exorciser or JFLAP'

Reply:
That implies that you do the thing by hand with the tool Exorsier or JFLAP; doing what you suggest is more for something like to say `By hand, using a pen or pencil`

Reply:
But I will try to make it a bit nicer